### DISCUSSION TITLE
Updating for  Magick.NET 13.6.0

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1601] [General]
When i install all libraries from Extra libraries i received this error (i take

 magick.core.dll from folder netstandard2.0)
![Uploading Screenshot_1.png…]()

When i take all libraries for version 13.5.0 - it is work ok

**Comment by jaka1-newbie** [comment]
I need version for netstandard2.0

**Comment by dlemstra** [comment]
I have no idea what kind of problem you are running into. Can you provide me with more information?
